azkaban/remote.py

```diff
@@ -518,6 +518,34 @@ def get_schedule(self, name, flow):
       )
     return res['schedule']
 
+  def sla(self, schedule, delay, rule='SUCCESS', emails=None, kill=True):
+    """Set schedule's SLA.
+
+    :param schedule: Schedule ID.
+    :param delay: SLA delay (format %H:%M, i.e. 00:45)
+    :param job: Job name (or empty to set SLA on flow).
+    :param rule: Status to check when duration ends `'SUCCESS'` or `'FINISHED'`
+    :param emails: Comma separated list of emails used for alerting
+    :param kill: Trigger the kill action
+    """
+    self._logger.debug('Setting SLA on schedule %s.', schedule)
+    send_email = 'true' if emails else 'false'
+    kill_workflow = str(kill).lower()
```

Comment:
```python
kill_workflow = 'true' if kill else 'false'
```

(This has the advantage that it works as expected if passed true-ish or false-ish objects.)